distributions/otelcol-contrib/manifest.yaml

```diff
@@ -97,6 +97,7 @@ processors:
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/groupbyattrsprocessor v0.106.1
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/groupbytraceprocessor v0.106.1
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sattributesprocessor v0.106.1
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/logdedupprocessor v0.106.1
```

Comment:
Note that it did not exist in this version yet as it was only introduced with https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/b92eb433024f91b3e73694571823bf0154f9e1f6 and is first released with v0.107.0, so in this version the build won't pass.